recipes/qtawesome/all/test_v1_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
I think that we could entirely remove this `test_v1_package`. New recipes are not required to have that folder anymore (see https://github.com/conan-io/conan-center-index/discussions/17406).

Reply:
Good catch!